src/theory/quantifiers/quant_conflict_find.cpp

```diff
@@ -649,6 +685,13 @@ bool QuantInfo::isMatchSpurious()
 
 bool QuantInfo::isTConstraintSpurious(const std::vector<Node>& terms)
 {
+  if (options().quantifiers.ievalMode != options::IevalMode::OFF)
+  {
+    // We rely on the instantiation evaluator. When the instantiation evaluator
+    // is enabled, this method (almost) always returns false. It may return
+    // true based on minor differences in the entailment tests.
```

Comment:
Clarify the consequences of this method returning `true` when the evaluator would have said `false`. Moreover what are the kind of differences that are at play here? Is it a matter of incompleteness?

Reply:
I don't concretely know the difference, the same general entailment test is being run, although by 2 very different pieces of code.